ophyd/tests/test_epicsmotor.py

```diff
@@ -4,60 +4,72 @@
 from copy import copy
 from numpy.testing import assert_allclose
 
-from ophyd import (EpicsMotor, Signal, EpicsSignal, EpicsSignalRO,
-                   Component as C, MotorBundle)
-from ophyd.utils.epics_pvs import (AlarmSeverity, AlarmStatus)
+from ophyd import (
+    EpicsMotor,
+    Signal,
+    EpicsSignal,
+    EpicsSignalRO,
+    Component as C,
```

Comment:
"Component as Cpt" convention flouted

Reply:
not a new flouting (this is just wrapping).  Early on we went back and forth of `C` or `Cpt` was clearer.

Reply:
Fair point that this is just wrapping, but in a bunch of places in `sim.py` you went through and changed `Component` to `Cpt`. Why?